package.json

```diff
@@ -57,7 +57,7 @@
     "@types/qs": "^6.5.3",
     "algoliasearch-helper": "^3.6.2",
```

Comment:
do not merge until this is updated based on

- https://github.com/algolia/algoliasearch-helper-js/pull/883